ddtrace/tracer/textmap.go

```diff
@@ -686,6 +686,11 @@ var (
 	// Equals character must be encoded with a tilde.
 	// Other disallowed characters must be replaced with the underscore.
 	originRgx = regexp.MustCompile(",|~|;|[^\\x21-\\x7E]+")
+
+	// validIDRegex is used to verify that the input is a valid hex string.
+	// The input must match the pattern from start to end.
+	// validIDRegex is applicable for both trace and span IDs.
+	validIDRegex = regexp.MustCompile("^[a-f0-9]+$")
```

Comment:
Nit for naming consistency 😅 (all regex vars are named `<something>Rgx` in this file)
```suggestion
	validIDRgx = regexp.MustCompile("^[a-f0-9]+$")
```

Reply:
Done